notebooks/introductory_infilling.ipynb

```diff
@@ -0,0 +1,2572 @@
x5cifBHgD9YCtwBtlHQu0wjgBamzm8wGam/l2Af8P+B3gBxwBepdQ3z7AVcC1oukYJ22xFWzfstpiu5leD0gG/mim/Q74CfgL4IZxJS23YFuV9D6W1SaltHFFyyyrHDvwHcYVwCbASWA30AGoCqwH/qdwHUp6r83HpaVdO94NOAS8ZMbVE8gCWhUqq0jbl/b65fnbKfQ+lLv+pbVlGXUuz3uwB+PvrXqhdirr2HK3hWyyyXb3biX8X9gRyAfamM/7AwnAefP/lVcL5D0GaOCCuXXB+H5fD2QCp4EvgTqFynwR2A+cBT4FqplpdYGlwCkzbSngVeDYEeb/dVnAUWBogbRR5vfBWWAV0KyEOnc1j69RYF8/8/95V6C2WZeYQsfdY+YZZT7/AlhWgbYuq24bgX9gnFtlAauBBgXSh2F8z2cCLxf33pn5XgOuYJwDXABGm+32TYE8GvA3H3+G47ncgxjfPeeAbUC7AmkdML4LszA6yuIo5jwD43vqsvk5ugB8Zu6PAn4wX3sjYCvuswi8CnxRIK3Y44CRwJIC+Q4BXxV4ngoEmY9bA2uAM8CPBd9fsw3+AyzHOK8qrl3dMTo+M4B04A3ApcDnciswzYzxCMbnbIQZw0lgeKHy3jAfNzA/C+fM2LYAVcy0F8yyssyYe1WkfQq060QgEfjFfN+qlVV2obpvxvjMXDTfz0FADyCtUDl/Ncu5aLaVB7DCjH8tULdA/s4Yn69zwF6gx43+nZsxjgMOAkeL+axXBd7G+H/rBPAhv55flastZLu9m9MDkE22271R8pf7d8DLJRyzB3i4tOPLyP/HAmn9gMNlHQv4Y3y5PQC4FUjvBBwrdMyLwKclvN5Q4OdSyvtDSekYVyXXmI+7AN8Cm4DYgjFVsC3+UCDtX8CH5uNw4DigCqRvo3wdPBVtk2ttXNEySy3HjKngF+oC4D8Fno8HFhWuQ0nvtZmntLRrx3cDfi74pWq+R68Wyluk7Ut7/fL87VC0g6dc9S+tLcuoc3neg1EltFNZx5a7LWSTTba7dyvu/0Jzf8ELRT2Athgdy+0wfhg9Yqb5YPyAci1wrD8QifFjqiHGj8Pphcos9mIRUB+IBmpgjIKZV+D/2poYnUytzOeeQKD5+BGMH/c2jE6avwPbSqn3gULfI7HXYqScF4swvqtGVqCtS6ybmb4ROAy0BKqbzyebaQEYP6zDzXadasZY7DkcRTsARlCODh4g2Pzu6AS4AMPN96sqFb+Q1APHDoCWGD/8I83j/2a+Z78r/FksGH9px2Fc3DiH8dn0NONLN4/zw+gEqGJ+dlIxOoRczXqeLvD5+Qyj8+N+M3+1YuqzCPjIfK1GGBeani7QvlfN13fB6Pw5Bnxgtt3vMTor7immzd/COIdxM7dugKL0C0Tlap8C7VrSBcliyy7h/bQ+MyW8v3bKf2GsCUZHZT+zvSPN5w25gb9zM8Y1Zj2rF44bY4mHxWZ6LWAJ8FZF20K227fJFC0hftUEo/cZpdQTSqk95nSXc0AbjF7qYpUjf2qBxz9hfGGUeqzW+hAwAeML6aRSKk4p1Rhj+G7ja/nNY17C+HIoTibQoJRhl6dLSfc006/F3VNr3R3jCsHD19kWBRcNvIRxdQ+MNknX5jdGgTLLo9Q2KSWmipZZnrY/UeDx5WKe30MhpbzXpaYV0BhI1VrnF6pHk0L5irR9OV+/Ispb/xLbsoyYyvMeFPx7K6jUY29BWwgh7i7HMX4EobXeqLXep7XO11onYnSGdC/pQK31Ia31Gq11jtb6FEZnROH872utU7XWZ4B/YkyPQmudqbVeoLW+pLXOMtMKHpsPtFFKVddaZ2itfzD3P43xQy1ZG9OS3gSClFLNSgjzc8xpWsqY3v0w5vQsjO/U07r46U0Z/HoeUN98Xi7lqBsYnfQHtNaXga+AIHP/o8BSrfVmrXUO8IrZFjfbk8BHWuvvtdZ5WutZQA7GaIvOGD98p2tjCvx8YEcFXnsQxoinNVrrXIyRFNUxRrpc13HaWFslC6OdumOM6EhXSrU2n28xzyceBOxa60+11le11rsxLtw8WqCceK31VvNznl0wAKWUB9AXmKCN5QFOYozWGVwg21Hz9fMwRsl4A6+bfwerMUZV+VNULsY5ajOzXbeY53J5GB0jAUopN621XWt9+Drb9X+11sfNv7cl/Pq5Kqns6/We1vqE1jodYwTM91rrBPMzuxCjsweMC7LLtdbLzfZeA+zE6PCBG/s7f0trfcb8G7IopRTG5/svZnqWefy19/Bmt4W4CaSDRwhAKXUfxo/hb8z/8D4BnsVYaK8OxlUzZWbXhY4tKz8YX1jXNMU4CSzzWK31HK11GMYPU40xtz0V4wuxToGtlta6H8X7FsjG6MEvKT0HGFioXjUxvpjXmbEcL/Af/1WKOUkqZ1uUJANoYn6ZXNO0HMdBKW1SRkwVLbOibV9uJbzXZaaZjgPeynGBz6YYQ5RvqOzCWcvzeuVUaluWElN53oOS4izz2Aq0hRBCFFbwQlEnpdQGc72vXzCmxJZ2oaiR2amcrpQ6jzGVqXD+Yi8WKaVqKKU+Ukr9ZB67GaijlHLRWl/E+DH7RyBDKbXM/CEPxv9z7xbo8D6D8d1Y+OLANZ8DEUqpJhg/8g9prRPMtPJeLMo0n5dLaXUrkK20C0dWm5ltkVnesiugGfB8oYsH3mb5N3LxCvN4K7/Z8ZJKye9ReY/bhDGaJNx8vBGjc6e7+fxavToVqtdQoODizyVdULl2vBvG5+7a8R9hjOS5pvBFILTWZV4YA/6NMSpltVLqiFJqknlseS/UlKddS/pcFVv2DajIhbHHCr0fYRjLINzo33lJ72NDjNFzuwocv9LcDze/LcRNIB084q6mlKqtlHoQYz70F1rrfRjDHDXGfG+UUiMxRnxccwJjCOs1ZeUHGKeU8lJK1cMYMTC3rGOVUq2UUj2VUlUxOmguY1yZ2A6cV8ZisNWVUi5KqTZmJ1URWutfMNYc+UA
```

Comment:
There is a part of this text that seems missing, and the formatting seems off (you can put "Recovering the data you put in") in the next cell already.

Reply:
ok